fetch_and_alert.py

```diff
@@ -1,158 +1,99 @@
-                                             cougars.at[idx, 'bbox2'], cougars.at[idx, 'bbox1'],
-                                             cougars.at[idx, 'bbox2'] + cougars.at[idx, 'bbox4'],
-                                             cougars.at[idx, 'bbox1'] + cougars.at[idx, 'bbox3'],
-                                             expansion=0,
-                                             use_normalized_coordinates=True,)
-                  imageBytes = BytesIO()
-                  img.save(imageBytes, format=img.format)
-                  smtp_server = smtp_setup(username, password, host)
-                  sendAlert(label, prob, imageBytes, smtp_server, username, to_emails)
-              # Write Dataframe to csv
-              cougars.to_csv(f'{log_dir}dataframe_{datetime.now().strftime("%m-%d-%Y_%H:%M:%S")}')
-
-
-def run_emails():
-    # Gets a list of attachments from unread emails from bigfoot camera
-    mail = imap_setup(host, username, password)
-    global timestamp
-    print('Starting Email Fetcher')
-    images = extractAttachments(fetch_emails(mail, from_emails, timestamp), mail, config_file)
-    print('Finished Email Fetcher')
-    print('Starting Detection')
-    detect(images)
-    print('Finished Detection')
+CHECKIN_INTERVAL = CONFIG['checkin_interval']
 
 
 def run_scraper():
+    ''''Gets the new images from strikeforce and runs a pre-trained
+    detector model on them and sends images with animals of interest to
+    specified emails'''
     print('Starting Web Scraper')
-    images = fetch_images(config_file)
+    images = fetch_image_api(CONFIG)
     print('Finished Web Scraper')
     print('Starting Detection')
-    detect(images)
+    detect(images, CONFIG)
     print('Finished Detection')
 
 
 def main():
+    '''Runs the fetching and alerting functions and begins scheduler
+    to loop the fetching and alerting periodically'''
     # Run the scheduler
     print("Running fetch_and_alert")
     # run_emails()
     run_scraper()
-    print("Sleeping since: " + str(datetime.now()))
+    print("Sleeping since: " + str(dt.now()))
 
 
 def checkin():
-    print("Checking in at: " + str(datetime.now()))
+    '''Sends server status to specified email at specified time interval'''
+    print("Checking in at: " + str(dt.now()))
     # Construct Email Content
     email_message = EmailMessage()
-    email_message.add_header('To', ', '.join(to_emails))
-    email_message.add_header('From', username)
+    email_message.add_header('To', ', '.join(TO_EMAILS))
+    email_message.add_header('From', USERNAME)
     email_message.add_header('Subject', 'Checkin')
     email_message.add_header('X-Priority', '1')  # Urgency, 1 highest, 5 lowest
     email_message.set_content('Still Alive :)')
     # Server sends email message
-    smtp_server = smtp_setup(username, password, host)
+    smtp_server = smtp_setup(USERNAME, PASSWORD, HOST)
     server = smtp_server
     server.send_message(email_message)
 
 
 main()
```

Comment:
This remains an unconventional way to use a function called main. Typically, main is the top-level function of the program and in Python you do a little work to make that be true.  When we are in the lab together, we can talk over what I think would be better here.  In essence, I think we should combine what is currently in `main` and `run_scraper` into a function called something like `fetch_detect_alert`, move the bare lines into a more conventional `main` and then have the conventional code that runs main if the module is executed at the bottom i.e. :

```python
if  __name__ == "__main__":
    main()
```